projects/hello_world/src/hello_world.c

```diff
@@ -0,0 +1,33 @@
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define INTERVAL 0.5
```

Comment:
It's not immediately clear from this what units `INTERVAL` is in - add a suffix like `INTERVAL_S` to tell readers that it's in seconds. (Also, there should be a blank line between the includes and any defines.)